test_scripts/API/VehicleData/common.lua

```diff
@@ -325,6 +325,24 @@ function m.getVDParams(pIsSubscribable)
   return out
 end
 
+--[[ @getSubVDParam: Return another available VD parameter for subscription
+--! @parameters:
+--! pParam: name of the VD parameter
+--! @return: another VD parameter
+--]]
+function m.getAnotherSubVDParam(pParam)
+  local out
+  for param, value in pairs(m.vd) do
+    if value ~= "" then
+      if param ~= pParam then
+        out = param
+        break
+      end
+    end
+  end
+  return out
+end
+
```

Comment:
I would change this function to the following:
```
  local params = m.getVDParams(true)
  local sortedParams = {}
  for k in m.spairs(params) do
    table.insert(sortedParams, k)
  end
  for i, p in pairs(sortedParams) do
    if p == pParam then
      if i == #sortedParams then
        return sortedParams[1]
      else
        return sortedParams[i+1]
      end
    end
  end
```
It allows to remove randomization of VD parameters combinations by returning next parameter after defined one.
If defined parameter is the latest one, function returns 1st.